### ISSUE TITLE
Option to close `List` menu

### ISSUE THREAD

**Issue by elpekenin** [magmax/python-inquirer#286]
I was wondering if there is an option such that, when an element of a `List` is selected, the list of elements is hiden and only the line with `Question: Answer` remains. 

Couldn't find anything on the documentation nor issues, but asking just in case. If it doesn't exist already, i could try making it myself, but i have no idea where/how to start, tbh

**Comment by Cube707** [comment]
no, all renderes will just break onto a new line upon completion and leave their rendered content on-screen

Thsi woul be a great genereal feature though and if implemented for the whole libary would be verry usefull in a lot of cases.

**Comment by Cube707** [comment]
Duplicate of #57
